package.json

```diff
@@ -23,54 +23,62 @@
-    "@types/jasmine": "^4.3.5",
-    "@types/jasmine-jquery": "^1.5.34",
-    "@types/jquery": "3.5.16",
-    "@types/karma": "6.3.4",
-    "@types/lodash": "4.14.196",
-    "@types/webpack": "5.28.1",
+    "@types/d3-hierarchy": "^2.0.0",
+    "@types/d3-selection": "^2.0.0",
+    "@types/d3-shape": "^2.0.0",
+    "@types/jasmine": "^3.6.2",
+    "@types/jasmine-jquery": "^1.5.33",
+    "@types/jquery": "3.5.5",
+    "@types/karma": "^6.3.4",
+    "@types/lodash": "4.14.166",
+    "@types/webpack": "4.41.25",
     "@typescript-eslint/eslint-plugin": "^6.2.1",
-    "coveralls": "3.1.1",
+    "core-js": "^3.32.0",
+    "coveralls": "^3.1.0",
+    "css-loader": "^5.0.1",
     "d3-hierarchy": "^3.1.2",
     "d3-selection": "^3.0.0",
     "d3-shape": "^3.2.0",
+    "eslint": "^8.43.0",
     "eslint-plugin-powerbi-visuals": "^0.8.1",
-    "jasmine": "5.1.0",
+    "jasmine": "^5.1.0",
     "jasmine-core": "^5.1.0",
+    "jasmine-jquery": "^2.1.1",
+    "jquery": "^3.7.0",
     "karma": "^6.4.2",
     "karma-chrome-launcher": "^3.2.0",
-    "karma-coverage": "2.2.1",
+    "karma-coverage": "^2.2.1",
     "karma-coverage-istanbul-reporter": "^3.0.3",
-    "karma-jasmine": "5.1.0",
+    "karma-jasmine": "^5.1.0",
     "karma-junit-reporter": "^2.0.1",
     "karma-remap-istanbul": "^0.0.2",
-    "karma-sourcemap-loader": "0.4.0",
-    "karma-typescript": "5.5.4",
-    "karma-typescript-preprocessor": "0.4.0",
-    "karma-webpack": "5.0.0",
-    "lodash-es": "4.17.21",
-    "playwright": "^1.36.2",
+    "karma-sourcemap-loader": "^0.3.8",
+    "karma-typescript": "^5.2.0",
+    "karma-typescript-preprocessor": "^0.4.0",
+    "karma-webpack": "^5.0.0",
+    "less": "^4.0.0",
+    "less-loader": "^7.2.1",
+    "lodash-es": "^4.17.20",
     "postcss-less": "^6.0.0",
-    "powerbi-models": "^1.12.6",
+    "powerbi-models": "^1.7.0",
     "powerbi-visuals-api": "~5.4.0",
-    "powerbi-visuals-tools": "5.0.2",
+    "powerbi-visuals-tools": "^5.0.1",
     "powerbi-visuals-utils-chartutils": "^6.0.1",
     "powerbi-visuals-utils-colorutils": "^6.0.1",
-    "powerbi-visuals-utils-dataviewutils": "^6.0.1",
+    "powerbi-visuals-utils-dataviewutils": "^2.4.0",
+    "powerbi-visuals-utils-formattingmodel": "^5.0.1",
     "powerbi-visuals-utils-formattingutils": "^6.0.1",
     "powerbi-visuals-utils-interactivityutils": "^6.0.2",
     "powerbi-visuals-utils-svgutils": "^6.0.1",
     "powerbi-visuals-utils-testutils": "^6.0.1",
-    "powerbi-visuals-utils-tooltiputils": "^6.0.1",
+    "powerbi-visuals-utils-tooltiputils": "^2.5.2",
     "powerbi-visuals-utils-typeutils": "^6.0.1",
-    "style-loader": "3.3.3",
-    "stylelint": "^15.10.2",
-    "ts-loader": "9.4.4",
-    "ts-node": "^10.9.1",
-    "typescript": "5.1.6",
-    "webpack": "^5.88.2"
+    "puppeteer": "^5.5.0",
+    "regenerator-runtime": "^0.13.7",
+    "style-loader": "^2.0.0",
+    "stylelint": "^14.16.1",
+    "ts-loader": "^8.0.12",
+    "ts-node": "^10.8.1",
+    "typescript": "^4.1.3"
```

Comment:
Please, update to 4.9.5

Reply:
Done: [81189d4](https://github.com/microsoft/powerbi-visuals-sunburst/pull/95/commits/81189d443a2bda4f576855676a9396fd4aace44a)